ERCS/ERC-xxxx

```diff
@@ -0,0 +1,16 @@
+---
+eip: xxxx
```

Comment:
To be assigned an ERC number a PR needs a specification as a minimum.

Reply:
updated